Week-2/Exercise 2.1/css/style.css

```diff
@@ -0,0 +1,268 @@
+*,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+:root {
+    --primary-color: rgb(145, 234, 122);
+    --secondary-color: rgb(181, 232, 169);
+}
+
+/*general styling */
+body {
+    font-family: "Poppins", sans-serif;
+    height: 100vh;
+    line-height: 1.6;
+    padding: 1em;
+}
+
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+    font-size: 1.3rem;
+}
+
+input,
+button {
+    font: inherit;
+}
+
+/*header styling*/
+
+.menu {
+    color: white;
+    cursor: pointer;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+}
+
+nav {
+    display: none;
+}
+
+nav a {
+    color: white;
+}
+
+nav>ul {
+    display: flex;
+    gap: 3rem;
+}
+
+ul>li:hover {
+    text-decoration: underline;
+}
+
+/* hero section styling */
+.hero-img {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: url(../images/hero-background.jpg);
+    background-size: cover;
+    /* background-position-x: 20%; */
+    width: 100%;
+    height: 90vh;
+    z-index: -1;
+    animation: introLoad 1s ease-in-out forwards;
+}
+
+section.hero {
+    color: white;
+    height: 90vh;
+    animation: moveDown 1s ease-in-out forwards;
+    opacity: 0;
+}
+
+section.hero>h1 {
+    font-size: 3rem;
+    margin-top: 2em;
+    line-height: 3.3rem;
+}
+
+.sub-heading {
+    font-size: 1.4rem;
+}
+
+.logo {
+    color: white;
+    font-weight: bold;
+}
+
+.arrow-down {
+    margin-top: 20rem;
+}
+
+/* articles section styling */
+
+section.articles * {
+    padding: 0;
+    margin: 0;
+    margin-top: 0.7rem;
+}
+
+section.articles {
+    margin-top: -8rem;
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 1fr;
+}
+
+article {}
```

Comment:
Never put up empty selectors

Reply:
Forgot to remove it. Will make sure that this won't happen again.